underfs/hdfs/pom.xml

```diff
@@ -69,7 +69,7 @@
         <dependency>
           <groupId>org.apache.hadoop</groupId>
           <artifactId>hadoop-core</artifactId>
-          <version>1.0.4</version>
+          <version>${hadoop.version}</version>
```

Comment:
I think it is different. https://github.com/Alluxio/alluxio/pull/5408/files aims to fix the test issue